Base/usr/share/man/man1/unzip.md

```diff
@@ -14,6 +14,8 @@ unzip will extract files from a zip archive to the current directory.
 
 The program is compatible with the PKZIP file format specification.
 
+The optional [files] argument can be used to include only specific files, with wildcards, in the unzip process. A `_` can be used as a single-character wildcard, and  `*` can be used as a variable-length wildcard.
```

Comment:
This is a surprisingly weird thing that unzip has, I had no idea it existed...

I guess it's fine to have it for compatibility :shrug:

Reply:
I'd also suggest being more clear that this filters the contents of the zip, not the files passed to unzip.

Reply:
Yea, it was listed as a want for #12303 so I wanted to whip it up real quick :^)

Reply:
New verbiage: "The optional [files] argument can be used to only extract specific files within the archive (using wildcards) during the unzip process."